### PR TITLE
Update fix for breaking API change

### PR DESCRIPTION
**Pull request by maier**
IMPROVEMENTS:

* Switch from deprecated circonus-gometrics api sub-package to the current [go-apiclient](https://github.com/circonus-labs/go-apiclient) package

BUG FIXES:

* Incorporate fix in [go-apiclient v0.6.3](https://github.com/circonus-labs/go-apiclient/releases/tag/v0.6.3) to address breaking change to `rule_set` CID format in public API
* incorporate fix in [go-apiclient v0.6.4](https://github.com/circonus-labs/go-apiclient/releases/tag/v0.6.4) graph.datapoint.alpha - doc:floating point number, api:string

### CODE REVIEW

**Comment by maier** [comment]
@appilon Hi! I'm new to this, so could you please give this a look to see if it is good to merge and ready for me to request a release build in the slack channel?

**Comment by appilon** [comment]
@maier I will try and take a look at this PR. Are you the maintainer?

File: .go-version [hunk 1]
```diff
@@ -1 +1 @@
-1.11.5
+1.13
```
Comment (appilon) [inline .go-version]: for reasons I can't recall off the top of my head, there were some minor reasons where we preferred v1.12.9 of Go over going to 1.13 at the moment. If you don't mind holding it back to that.

File: .travis.yml [hunk 2]
```diff
@@ -4,7 +4,7 @@ services:
 - docker
 language: go
 go:
-  - "1.11.x"
+  - "1.13.x"
```
Comment (appilon) [inline .travis.yml]: same as above

**Comment by appilon** [comment]
@maier there is a lot happening in this PR, and unfortunately the TeamCity setup has many tests failing due to a "bad broker". Could we try and do this in a few PRs, maybe with the api library change as the first PR. Then switch to terraform v0.12.9 (current latest release of core), and then finally a switch to the standalone plugin SDK (which will probably go pretty smoothly), so we can isolate potential problems between large changesets.

**Comment by appilon** [comment]
I'm not familiar with the provider or backend, I can try and assist in getting TeamCity more green.

**Comment by maier** [comment]
@appilon, yes, I'm the new maintainer. Where can I find out more about TeamCity setup? I can try to get the tests to work correctly.

**Comment by maier** [comment]
@appilon will close and start over with smaller PRs.

**Pull request closed without merging**